Source/CDRRunState.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, CedarRunState) {
+    CedarRunStateNotYetStarted = 0,
+    CedarRunStateBuildingTree  = 1,
```

Comment:
I mostly wanted to clarify since "building" could be overloaded in the context of Xcode, building projects and targets and whatnot.

Looking at it again... I wonder if there's a better phrase that would indicate users what is actually happening here. Perhaps `CedarRunStatePreparingToRunTests` ?